### PR TITLE
#7 create a command to erase the scoreboard

### PR DESCRIPTION
**Pull request by nakool**

### CODE REVIEW

File: src/game [hunk 1]
```diff
@@ -8,7 +8,17 @@ import typer
 import moreorless
 
 app = typer.Typer()
-SCOREBOARD_PATH = "scoreboard.json"
+SCOREBOARD_PATH = "src/scoreboard.json"
```
Comment (miragecentury) [inline src/game]: Better use a path outside of the project. like /tmp/more_or_less/local_scoreboard.json

File: src/game [hunk 2]
```diff
@@ -8,7 +8,17 @@ import typer
...
+
+
+@app.command()
+def erase_scoreboard(scoreboard_path: pathlib.Path = SCOREBOARD_PATH):
+    """
+    More or less Erase Scoreboard
+    """
+    moreorless.erase_player_score(player_name=str,
+                                  player_score=int,
```
Comment (miragecentury) [inline src/game]: The objective is to erase all the scoreboard, does not need for player_name or score ?

File: src/moreorless.py [hunk 3]
```diff
@@ -11,6 +11,34 @@
...
+        "player": player_name,
+        "score": player_score
+    }
+    for _i[str] in _scoreboard:
+        _scoreboard.clear()
+        print(f"The scoreboard {scoreboard_path=} is erased!")
+        with open(scoreboard_path, "w") as _ef:
+            json.dump(_scoreboard, _ef)
+
+
```
Comment (miragecentury) [inline src/moreorless.py]: Need rework and miss the objective.
Objective is to empty the full scoreboard.
Actions : 
- [x] If the scoreboard file exist delete it.
- [x] Create a new scoreboard file empty of score

**Review by miragecentury — CHANGES_REQUESTED** [review]
Missing the Objective (maybe not enough clear).
Missing Test

File: src/game (hunk 2 above)
Comment (nakool) [reply src/game]: It's updated now by removing 'player_name and player_score' parameters

File: src/game (hunk 1 above)
Comment (nakool) [reply src/game]: It's fix now by replacing the path by a fix path outside the game

File: src/moreorless.py (hunk 3 above)
Comment (nakool) [reply src/moreorless.py]: It's work now

**Pull request merged**